src/Utils.js

```diff
@@ -1,3 +1,78 @@
 (function(){
   LifeGamification.utils = {};
+  LifeGamification.work = {};
+  LifeGamification.refreshTimer;
+
+  LifeGamification.utils.setStartTime = function () {
+    return new Promise((resolve, reject) => {
+      LifeGamification.repository.getWork()
+      .then(function(result){
+        if(result){
+          LifeGamification.work = result;
+          resolve();
+        }
+      });
+    });
+  }
+
+  LifeGamification.utils.startTiming = function () {
+    if(!($('.timer__select-skill').val())){
+      console.log("Error: Add a skill first.");
+      return;
+    }
+    let currentTime = new Date().getTime();
+    let skillName = $('.timer__select-skill').val();
+    LifeGamification.repository
+      .setWork({startTime: currentTime, name: skillName})
+    .then(LifeGamification.utils.handleTimer());
+  }
+
+  LifeGamification.utils.endTiming = function () {
+    const skill = LifeGamification.
+      skillsCollection[LifeGamification.work.name];
+    const expToAdd = Math.floor(calcTime() / 60);
+    LifeGamification.models.updateExp(skill, expToAdd);
+
+    LifeGamification.repository.setWork({});
+    clearInterval(LifeGamification.refreshTimer);
+    LifeGamification.view.finishTimer();
+  }
+
+  displayTime = function(time) {
+    let hours = Math.floor(time / 3600);
+    time %= 3600;
+    let minutes = Math.floor(time / 60);
+    time %= 60;
+    let seconds = time;
+    if(hours < 10){
```

Comment:
I'm pretty sure that there is an implemented mechanizm for formatting time in js.

Reply:
In this article on stackoverflow there isnt: https://stackoverflow.com/questions/18229022/how-to-show-current-time-in-javascript-in-the-format-hhmmss.
I found a library which formats dates, but it doesn't format to HH:MM:SS.

Reply:
Did you tried:
```javascript
moment.format("HH:mm:ss")
``` 

??